src/main/java/de/blazemcworld/blazinggames/items/contexts/EmptyItemContext.java

```
@@ -16,6 +16,19 @@
 
 package de.blazemcworld.blazinggames.items.contexts;
 
+import org.bukkit.entity.Player;
+
+import java.text.ParseException;
+
 public class EmptyItemContext implements ItemContext {
     public static EmptyItemContext instance = new EmptyItemContext();
+
+    public static EmptyItemContext parse(Player player, String string) throws ParseException {
+        if(string.isBlank())
+        {
+            return instance;
+        }
+
+        throw new ParseException("Do mention that this item's context is empty.", string.length());
```

is there any point to throwing an exception rather than just always returning an empty instance?

---

is there any reason to allow providing data to contextless items?